templates/_templating_scripting.py

```diff
@@ -277,6 +277,15 @@ def create_issue(title: str, body: str) -> None:
                 continue
 
 
+def check_dir(args: argparse.Namespace) -> None:
+    # Ensures templates/ dir aligns with _templating_config.json.
+
+    changed_templates = [Path(TEMPLATES_DIR, template_name) for template_name in TEMPLATES_DIR.rglob("*")]
```

Comment:
`changed_templates` is a misleading name. I'm guessing this is legacy from the original code?

Reply:
https://github.com/SciTools/.github/pull/61/commits/e47d9e12240dbc91dc530b84ef0c787516d86ee8